backend/backends/qwen_custom_voice_backend.py

```
@@ -100,22 +107,27 @@ def _load_model_sync(self, model_size: str) -> None:
-                self.model = Qwen3TTSModel.from_pretrained(
-                    model_path,
-                    torch_dtype=torch.float32,
-                    low_cpu_mem_usage=False,
-                )
-            else:
-                self.model = Qwen3TTSModel.from_pretrained(
-                    model_path,
-                    device_map=self.device,
-                    torch_dtype=torch.bfloat16,
-                )
+            with force_offline_if_cached(is_cached, model_name):
```

Ruff (0.15.10)

[error] Undefined name `force_offline_if_cached`

(F821)